packages/pancake-uikit/src/components/Svg/Icons/Github.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from "react";
+import Svg from "../Svg";
+import { SvgProps } from "../types";
+
+const Icon: React.FC<SvgProps> = (props) => {
+  return (
+    <Svg xmlns="http://www.w3.org/2000/svg" {...props}>
```

Comment:
I think we ought to declare the viewbox value here. Not sure we need the xmlns attribute? Goes for the other SVGs too